tensorflow_full_runtime.supp

```diff
@@ -9909,6 +9229,24 @@
    fun:_ZN6google8protobuf16RepeatedPtrFieldIN10tensorflow24KernelDef_AttrConstraintEE9MergeFromERKS4_
    fun:_ZN6google8protobuf16RepeatedPtrFieldIN10tensorflow24KernelDef_AttrConstraintEEC1ERKS4_
 }
+{
+   <tensorflow_full>
```

Comment:
Just that? Did you spent time improving the list or is it still WIP ?

Reply:
Yeah most of these suppression entries were equivalent except for the last frame which was specific to a TF op. So I just introduced a `...` frame above which captures all of them. In fact it may even capture this suppression now that I'm looking at it.